image-loader/src/androidUnitTest/kotlin/com/seiko/imageloader/screenshot/ComposeScreenShotTest.kt

```diff
@@ -0,0 +1,122 @@
+package com.seiko.imageloader.screenshot
+
+import androidx.activity.ComponentActivity
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.ColorPainter
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onRoot
+import androidx.compose.ui.unit.dp
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.takahirom.roborazzi.RoborazziRule
+import com.seiko.imageloader.ImageLoader
+import com.seiko.imageloader.intercept.Interceptor
+import com.seiko.imageloader.model.ImageResult
+import com.seiko.imageloader.rememberImagePainter
+import kotlinx.coroutines.delay
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.GraphicsMode
+
+@RunWith(AndroidJUnit4::class)
+@GraphicsMode(GraphicsMode.Mode.NATIVE)
+@Config(sdk = [32])
+class ComposeScreenShotTest {
+
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<ComponentActivity>()
+
+    @get:Rule
+    val roborazziRule = RoborazziRule(
+        composeRule = composeTestRule,
+        captureRoot = composeTestRule.onRoot(),
+        options = RoborazziRule.Options(
+            outputDirectoryPath = "src/androidUnitTest/snapshots/images",
```

Comment:
For your reference, the default captureType has been changed to None in alpha-3, so you have to specify CaptureType.LastImage here if you update alpha-3 to capture the last image.